content/plugins/ariaieboy-currency.md

```diff
@@ -0,0 +1,15 @@
+---
+name: Currency
+slug: ariaieboy-currency
+author_slug: ariaieboy
+categories: [table-column]
+description: Bring back laravel-money formatter for Filament Text Column.
+discord_url: 
+docs_url: https://github.com/ariaieboy/filament-currency/blob/main/README.md
```

Comment:
Please fix this so that it points to a raw markdown file URL, like `https://raw.githubusercontent.com/filamentphp/spatie-laravel-media-library-plugin/3.x/README.md` in our example.